### PR TITLE
Feature/contact us service

### PR DESCRIPTION
**Pull request by rusiruavb**
## Changes in this PR
- Email handling service
- Contact us service functions
- Contact us controller functions
- Contact us email template

### CODE REVIEW

File: src/api/controllers/Contact.controller.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const createContact = async (request: Request, response: Response, next:
   await ContactService.insertContact(request.body)
     .then((data) => {
       // Send email 
-      let emailTemplate = path.join(__dirname, '../..', 'templates', 'Contact-Us-Email-Template.html');
+      let emailTemplate = 'Contact-Us-Email-Template.html';
```
Comment (senuravihanjayadeva) [inline src/api/controllers/Contact.controller.ts:19]: why we are not using const instead of let

File: src/api/controllers/Contact.controller.ts [hunk 2]
```diff
@@ -26,12 +26,18 @@ export const createContact = async (request: Request, response: Response, next:
         date_time: moment(data.createdAt).format('LLL'),
```
Comment (senuravihanjayadeva) [inline src/api/controllers/Contact.controller.ts:26]: can we use dateTime instead of date_time

File: src/api/controllers/Contact.controller.ts [hunk 3]
```diff
@@ -40,14 +46,36 @@ export const createContact = async (request: Request, response: Response, next:
...
+}
+
+/**
+ * @param {Request} request - Request from the frontend
+ * @param {Response} response - Response that need to send to the client
+ * @param {NextFunction} next - Next function
+ * @returns {IContact[]} Removed contact information
+ */
+export const removeContact = async (request: Request, response: Response, next: NextFunction) => {
+  await ContactService.archiveContact(request.params.id)
```
Comment (senuravihanjayadeva) [inline src/api/controllers/Contact.controller.ts:73]: To maintain the structure please rename this as contactId  instead of id

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

**Comment by rusiruavb** [comment]
@senuravihanjayadeva changes are applied

**Review by senuravihanjayadeva — APPROVED** [review]

**Pull request merged**